packages/validator/templates/OAS2OIS.json

```diff
@@ -0,0 +1,288 @@
+			}
+		}
+	],
+	"info": {
+		"title": {
+			"__actions": [
+				{
+					"__copy": {
+						"__target": "title"
+					}
+				}
+			]
+		},
+		"version": {
+			"__actions": [
+				{
+					"__copy": {
+						"__target": "version"
+					}
+				}
+			]
+		},
+		"__ignore": {}
+	},
+	"servers": {
+		"__arrayItem": {
+			"url": {
+				"__regexp": "^(https?|ftp)://[^\\s/$.?#].[^\\s]*$",
+				"__actions": [
+					{
+						"__copy": {
+							"__target": "apiSpecifications.servers[0].url"
+						}
+					}
+				]
+			}
+		}
+	},
+	"paths": {
+		"__keyRegexp": "^\\/[^\\s'\"\\\\]+$",
+		"__objectItem": {
+			"__keyRegexp": "^(get|post)$",
+			"__objectItem": {
+				"__actions": [
+					{
+						"__insert": {
+							"__target": "apiSpecifications.paths.{{1}}.{{2}}.parameters",
+							"__value": []
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[].name",
+							"__value": "{{1}}"
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[_].operation.method",
+							"__value": "{{2}}"
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[_].operation.path",
+							"__value": "{{1}}"
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[_].reservedParameters",
+							"__value": [
+								{
+									"name": "_type"
+								},
+								{
+									"name": "_path"
+								},
+								{
+									"name": "_times"
+								}
+							]
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[_].fixedOperationParameters",
+							"__value": []
+						}
+					}
+				],
+				"parameters": {
+					"__arrayItem": {
+						"name": {
+							"__actions": [
+								{
+									"__copy": {
+										"__target": "apiSpecifications.paths.{{1}}.{{2}}.{{3}}.name"
+									}
+								},
+								{
+									"__copy": {
+										"__target": "endpoints[_].parameters[].name"
+									}
+								},
+								{
+									"__copy": {
+										"__target": "endpoints[_].parameters[_].operationParameter.name"
+									}
+								}
+							]
+						},
+						"in": {
+							"__regexp": "^(path|query|header|cookie)$",
+							"__actions": [
+								{
+									"__copy": {
+										"__target": "apiSpecifications.paths.{{1}}.{{2}}.{{3}}.in"
+									}
+								},
+								{
+									"__copy": {
+										"__target": "endpoints[_].parameters[_].operationParameter.in"
+									}
+								}
+							]
+						},
+						"schema": {
+							"__conditions": [
+								{
+									"__if": {
+										"default": ".*"
+									},
+									"__then": {
+										"default": {
+											"__actions": [
+												{
+													"__copy": {
+														"__target": "endpoints[_].parameters[_].default"
+													}
+												}
+											]
+										}
+									}
+								}
+							],
+							"__ignore": {}
+						},
+						"__ignore": {}
+					}
+				},
+				"__ignore": {}
+			}
+		}
+	},
+	"components": {
+		"schemas": {
+			"securitySchemes": {
```

Comment:
I think this might be an error?

According to OAS specs, `schemas` and ` securitySchemes` should be on the same level in the Components Object.

Reply:
See: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#components-object

Reply:
Good catch, no need for `schemas` there

Reply:
Replaced in 1b7124c